taxi/urls.py

```diff
@@ -46,6 +51,22 @@
     path(
         "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
     ),
+    path("drivers/create/", DriverCreateView.as_view(), name="driver-create"),
+    path(
+        "drivers/<int:pk>/license-update/",
+        DriverLicenseUpdateView.as_view(),
+        name="driver-update"
+    ),
+    path(
+        "drivers/<int:pk>/update",
+        DriverUpdateView.as_view(),
+        name="driver-full-update"
```

Comment:
yes, they differ, since the URLs lead to different views, which in turn use different forms. The license update page contains only the license number field, and the driver update page contains all the fields that are associated with it

Reply:
Agree, this is redundant